app/Jobs/Job.php

```diff
@@ -20,4 +21,9 @@ abstract class Job implements ShouldQueue
     |
     */
     use InteractsWithQueue, Queueable, SerializesModels;
+
+    public function backoff(): array
+    {
+        return Config::get('queue.backoff');
```

Comment:
Do you have any idea what happens in case this is an empty array? Will it just retry immediately?

Reply:
Just tried this, yes